src/display/api.js

```diff
@@ -68,13 +68,39 @@ var isWorkerDisabled = false;
 var workerSrc;
 var isPostMessageTransfersDisabled = false;
 
-//#if PRODUCTION && !SINGLE_FILE
-//#if GENERIC
-//#include $ROOT/src/frameworks.js
-//#else
-//var fakeWorkerFilesLoader = null;
-//#endif
-//#endif
+var fakeWorkerFilesLoader = null;
+var useRequireEnsure = false;
+if (typeof PDFJSDev !== 'undefined' &&
+    PDFJSDev.test('GENERIC && !SINGLE_FILE')) {
+  // For GENERIC build we need add support of different fake file loaders
+  // for different  frameworks.
+  if (typeof window === 'undefined') {
```

Comment:
With this code being inlined here, do we still need to keep https://github.com/mozilla/pdf.js/blob/master/src/frameworks.js around?